builtin/assign_osh.py

```diff
@@ -19,41 +19,34 @@
 from core import vm
 from data_lang import j8_lite
 from display import ui
-from frontend import flag_util
-from frontend import args
+from frontend import args, consts, flag_util
```

Comment:
I merged those two lines, but was there a reason that those were separated?

Reply:
I usually put each FILE on a seprate line, but if there are names within a file, those go on the same line

I don't actually remember why I do that -- I think before auto-formatting it was a bit easier to maintain.

This other style is shorter, which may be nice since we count lines :-)